Code/MinimalLib/tests/tests.js

```diff
@@ -13,7 +13,11 @@ const assert = require('assert');
 const {
     performance
   } = require('perf_hooks');
-var initRDKitModule = require("../demo/RDKit_minimal.js");
+const legacySuffix = process.env.RDK_MINIMAL_LIB_SUPPORT_LEGACY_BROWSERS === 'ON' ? '_legacy' : '';
+if (legacySuffix) {
```

Comment:
Same story, please put this in a separate file

Reply:
Sorry, but having to update two files whenever we add a test seems really inconvenient to me and I would rather not do that.
How about changing the code from
```javascript
const legacySuffix = process.env.RDK_MINIMAL_LIB_SUPPORT_LEGACY_BROWSERS === 'ON' ? '_legacy' : '';
if (legacySuffix) {
    console.log(`Loading legacy plain JS MinimalLib`);
}
var initRDKitModule = require(`../demo/RDKit_minimal${legacySuffix}.js`);
```
to:
```javascript
const minimalLibName = process.env.RDKIT_MINIMAL_JS || 'RDKit_minimal.js';
const minmalLibPath = '../demo/' + minimalLibName;
console.log('Loading ' + minmalLibPath);
var initRDKitModule = require(minmalLibPath);
```
I think that makes it tidier and avoids duplicating the files.

Reply:
I don't think we'd need to duplicate the entire file.  My idea would be to have testes.js cover as much as possible using the wasm version and then to have something for the legacy-mode which does a very basic test (no need to repeat the full set of tests that the wrappers work when they're being generated from the same code using the same compiler).

But we can leave it if you really want things this way

Reply:
it is true they are generated by the same compiler, but compiling to different languages. It would be harder to figure out something is broken in the plain JS library if the tests do not run. My code change proposal above only allows to override the `RDKit_minimal.js` location through an environment variable - it is a really small change.